pkg/build/build.go

```diff
@@ -79,6 +79,7 @@ func getBuilder(targetOS, targetArch, vmType string) (builder, error) {
 		{"openbsd", "amd64", []string{"gce", "vmm"}, openbsd{}},
 		{"netbsd", "amd64", []string{"gce", "qemu"}, netbsd{}},
 		{"freebsd", "amd64", []string{"gce", "qemu"}, freebsd{}},
+		{"testos", "testarch", []string{"qemu"}, testBuilder{}},
```

Comment:
We already have "test" OS and "64" arch in it, it would be reasonable to use it here. If this code will try to e.g. obtain sys/targets target for OS/arch later, it won't fail with unknown arch. See sys/targets/targets.go for all targets.